.env.example

```
@@ -2,4 +2,9 @@ POSTGRES_DB=postgres://user:password@postgres-test:5432/paymentcodedb?sslmode=di
 POSTGRES_DB_TEST=postgres://user:password@localhost:54320/paymentcodedb?sslmode=disable
 LOCAL_POSTGRES_DB=postgresql://postgres:postgres@localhost:5432/test?sslmode=disable
 REDIS_HOST=redis:6379
-REDIS_HOST_TEST=localhost:6379
\ No newline at end of file
+REDIS_HOST_TEST=localhost:6379
+AWS_REGION=ap-southeast-2
+AWS_ENDPOINT = localstack:4566
+AWS_ENDPOINT_TEST= http://localhost:4566
+AWS_SQS_URL = localstack:4566/000000000000/payment-queue
```

a bit typo, no need space before/after `=`
```suggestion
AWS_ENDPOINT=localstack:4566
AWS_ENDPOINT_TEST=http://localhost:4566
AWS_SQS_URL=localstack:4566/000000000000/payment-queue
```

---

thanks :pray: